pontis-ui/src/components/FAQ.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const faqs = [
+  {
+    question: "Does Pontis only provide financial data?",
+    answer:
+      "Pontis is starting with financial data but will quickly expand to sports, weather, news — any event that happens anywhere should be available on-chain.",
+  },
+  {
+    question: "What's the best thing about logo?",
+    answer:
+      "I don't know, but the square is a big plus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas cupiditate laboriosam fugiat.",
+  },
+  {
+    question: "What's the best thing about the square?",
+    answer:
+      "I don't know, but the corners are a big plus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas cupiditate laboriosam fugiat.",
+  },
+];
+
+const FAQ = () => (
+  <div className="w-full max-w-7xl divide-y-2 divide-slate-200">
+    <h2 className="text-3xl font-extrabold text-slate-900">
+      Frequently asked questions
```

Comment:
Take a look at the FAQ header at the bottom of the `features` page. I could also turn it into the `Heading` component used elsewhere. Thoughts?

Reply:
Reusing the `Heading` component seems useful here - why is there a style diff between this and `Heading`?

Reply:
No real reason. I was just trying different things